### PR TITLE
Make stacks show how much there are in inventory

### PR DESCRIPTION
**Pull request by SandPoot**
## About The Pull Request
Items that stack will always show their quantity if not on a /turf, such as being inside a bag who is on the floor.
This
***From [TGMC/8566](https://github.com/tgstation/TerraGov-Marine-Corps/pull/8566)***

## Why It's Good For The Game
Easier inventory management.

## Changelog
:cl:
qol: stacks now show how many items there are when in your inventory.
/:cl:

### CODE REVIEW

**Review by deathride58 — APPROVED** [review]

File: code/game/objects/items/stacks/stack.dm [hunk 1]
```diff
@@ -132,6 +132,14 @@
 	else
 		icon_state = "[initial(icon_state)]_3"
 
+/obj/item/stack/update_overlays()
+	. = ..()
+	if(isturf(loc))
+		return
```
Comment (BlueWildrose) [inline code/game/objects/items/stacks/stack.dm:138]: Doesn't ore stacks make use of overlays for their own stack system? Wouldn't this interfere with that?

**Review by BlueWildrose — COMMENTED** [review]
Just a minor concern I got.

File: code/game/objects/items/stacks/stack.dm (hunk 1 above)
Comment (SandPoot) [reply code/game/objects/items/stacks/stack.dm:138]: This is a legitimate issue but also storage code sucks, i'm stumped and i hate it.

Comment (deathride58) [reply code/game/objects/items/stacks/stack.dm:138]: This should probably just be `return .`

From there that should Just Work

```suggestion
		return .
```

Comment (SandPoot) [reply code/game/objects/items/stacks/stack.dm:138]: Not actually a problem, `return` always returns what's on `.`

**Comment by SandPoot** [comment]
Reminder that this isn't good only because of storages that show numbers for stacks.

**Review by deathride58 — APPROVED** [review]

**Pull request merged**